src/@next/components/molecules/CreditCardNumber/CreditCardNumber.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { CreditCardIcon } from "../../atoms";
+
+import { IProps } from "./types";
+
+export const CreditCardNumber: React.FC<IProps> = ({
+  provider,
+  last4Digits
+}) => {
+  return (
+    <>
```

Comment:
Consider wrapping this element in a div to avoid having parent styles affecting how it's displayed (like flexbox)